h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -599,7 +599,7 @@ def run_tests(tests, run_in_isolation=True):
     #flatten in case of nested tests/test suites
     all_tests = reduce(lambda l, r: (l.extend(r) if isinstance(r, (list, tuple)) else l.append(r)) or l, tests, [])
     for test in all_tests:
-        if not(hasattr(test, 'tag') and (('H2OANOVAGLM' in test.tag) or ('H2OMaxRGLM' in test.tag))): # exclude AnovaGLM because it does not have score function
+        if not(hasattr(test, 'tag') and (('H2OInfogram' in test.tag) or ('H2OANOVAGLM' in test.tag) or ('H2OMaxRGLM' in test.tag))): # exclude anovaglm, infogram for it has no scoring function
```

Comment:
hackme! this is not the right way to disable infogram integration to `sklearn`, this is just disabling a set of tests, so that we end up with ignored tests and a broken integration.

Please have a look at https://github.com/h2oai/h2o-3/pull/5779

Reply:
Yes, I approved the PR and hopefully, you will merge it and I will use it.

Reply:
I incorporated #5779.